metadata_service/proxy/atlas_proxy.py

```diff
@@ -93,43 +94,78 @@ def _get_table_entity(self, *, table_uri: str) -> Tuple[EntityUniqueAttribute, D
                 table_info['entity'],
                 qualifiedName=table_info.get('name')), table_info
         except Exception as ex:
-            LOGGER.exception('Table not found. {}'.format(str(ex)))
+            LOGGER.exception(f'Table not found. {str(ex)}')
```

Comment:
didn't know about that before. just read (https://cito.github.io/blog/f-strings/). pretty neat!

Reply:
yes, f strings are really cool :)